Juris.js

```diff
@@ -249,32 +253,32 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "journalArticle",
+				"title": "Der Schutz der äußeren Sicherheit Deutschlands durch das Strafrecht",
 				"creators": [
 					{
 						"firstName": "Nina",
 						"lastName": "Nestler",
 						"creatorType": "author"
 					}
 				],
-				"notes": [
+				"date": "2013",
+				"issue": "125",
+				"journalAbbreviation": "ZStW",
+				"libraryCatalog": "Juris",
+				"pages": "259-298",
+				"publicationTitle": "Zeitschrift für die gesamte Strafrechtswissenschaft, Wenn Sie den Link markieren (linke Maustaste gedrückt halten) können Sie den Link mit der rechten Maustaste kopieren und in den Browser oder in Ihre Favoriten als Lesezeichen einfügen.",
```

Comment:
Can you fix the publication title here? The text ", Wenn Sie den Link markieren (linke Maustaste gedrückt halten) können Sie den Link mit der rechten Maustaste kopieren und in den Browser oder in Ihre Favoriten als Lesezeichen einfügen." should be deleted. Maybe the xpath for it can be improved.